gadget/layout.go

```diff
@@ -89,6 +83,13 @@ type LaidOutStructure struct {
 	ResolvedContent []ResolvedContent
 }
 
+// IsRoleMBR returns whether a structure's role is MBR or not.
+// meh this function is weirdly placed, not sure what to do w/o making schemaMBR
+// constant exported
+func IsRoleMBR(ls LaidOutStructure) bool {
```

Comment:
it's probably a good idea to test/use this in TestGadgetImplicitRoleMBR

Reply:
done, required a bit of fiddling due to function signatures, but this is tested there